app/models/group.rb

```diff
@@ -0,0 +1,5 @@
+class Group < ApplicationRecord
```

Comment:
Let's simply if and have [array field](https://guides.rubyonrails.org/active_record_postgresql.html#array) inside user for the groups the user belongs to and a list of required groups in the challenge